VitaminJProcessing.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  4 02:05:08 2024
+
+@author: Anupama Rajendra
+"""
+import csv
+import numpy as np
+import pandas as pd
+
+#The purpose of this code is to take the flux vs. energy data derived from OpenMC and
+# restructure it into a form appropriate for ALARA.
+
+#Loading the csv that contains the energy bounds of the Vitamin J structure 
+Vit_J = 'VitaminJEnergyGroupStructure.csv'
+Vit_J_Groups = pd.read_csv(Vit_J, header=1)
+x_lh = Vit_J_Groups.iloc[:, 1]
+
+x = sorted(x_lh, reverse=True) #Energy bounds arranged from high energy to low energy
+        
+#Loading the csv file that contains the energy bins and neutron flux values (from OpenMC)
+#This csv file was created in a Python code that reads the h5 output from OpenMC
+Flux = 'Neutron_Flux.csv'
+Flux_Data = pd.read_csv(Flux, header=1)
+# Extract both columns from the CSV file
+y_lh = Flux_Data.iloc[:, 0]
+y = sorted(y_lh, reverse=True) #Energy bins arranged from highest to lowest energy
+z_lh = Flux_Data.iloc[:, 1]
+z = sorted(z_lh, reverse=True) #Fluxes arranged to highest to lowest corresponding energy
+#This section adds up the neutron fluxes between each pair of energy bounds in
+#the Vitamin J structure
+
+#Initializing the array of fluxes that are summed up
+flux_sums = [0] * (len(x) - 1)
+
+#Iterating over each interval of energies in the Vit J structure
```

Comment:
Notwithstanding the fact that you can avoid the whole thing by making OpenMC group structure match Vitamin-J....

This nested loop approach is a little cumbersome.  I'd recommend that you do something like:
```
bin = 0

for result_ebound, result_flux in results:
    if result_ebound > ebounds[bin]:
        bin +=1
    flux[bin] += result_flux
```

Reply:
Approached this using a for-while-if statement